website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
 
 :::info "User access" is not "Model access"
 
-**User groups and access** and **model groups and access** mean two different things. "Model groups and access" is a specific term used in the language of dbt-core. Refer to [Model access](/docs/collaborate/govern/model-access)  for more info on what it means in dbt-core.
+**User groups and access** and **model groups and access** are two different things. "Model groups and access" is a specific term used in the language of [model governance](/docs/collaborate/govern/about-model-governance). Refer to [Model access](/docs/collaborate/govern/model-access) for more information.
```

Comment:
Since we're using it on only these two pages and the info is unlikely to change, maintaining a reusable might be more work. I will add the more specific language from the model access page to this page. I like the specificity.

Reply:
What if we change the perspective on both but keep similar information?

"You're on this page to do x which is not y'
"You're on this page to do y, which is not x"

Reply:
I love that @matthewshaver. Feels more appropriate